smarta/smarta/StationLinesTableViewController.swift

```diff
@@ -108,6 +111,20 @@ class StationLinesTableViewController: UITableViewController {
         }
     }
     
+    func searchBar(_ searchBar: UISearchBar, textDidChange searchText: String) {
+        if searchText.count == 0 {
```

Comment:
I think you should be able to use this instead here.

```
        if searchText.isEmpty {
```

Reply:
ah perfect!